.github/ISSUE_TEMPLATE/convergence_epic.md

```diff
@@ -1,48 +1,55 @@
 ---
-name: (internal) Component convergence epic
-about: (team member use only) Epic issue tracking convergence of a particular component
+name: (internal) Component implementation epic
+about: (team member use only) Epic issue tracking implementation of a particular component
 ---
 
 <!--
 These issues are used by core contributors to track the list of items that should be
-completed as part of converging a component. More info can be found here: https://github.com/microsoft/fluentui/wiki/Component-Convergence-Guide
+completed as part of creating a v9 component. More info can be found here: https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide
 -->
 
-## Preparation:
+## Preparation
 
-- [ ] Open UI Research
+- [ ] [Open UI Research](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#open-ui-research)
   - [link to https://open-ui.org/]
-- [ ] Open GitHub issues related to component
+- [ ] [Open GitHub issues related to component](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#find-open-issues-on-github)
   - [link to each issue]
-- [ ] react-\* package scaffolded with the right ownership in CODEOWNERS
-  - [link to package / PR]
-- [ ] Component Spec authored and reviewed
-  - [link to spec in component package / PR]
+- [ ] [Create react-\* package and component from template](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#component-package)
+  - [link to PR]
+  - [link to package: `https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-<your-component>`]
+- [ ] (Optional) [Draft implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation)
+  - [link to PR, if applicable]
+- [ ] [Component Spec authored](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#component-spec) and [reviewed](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#spec-review)
+  - [link to PR]
+  - [link to Spec.md: `https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-<your-component>/Spec.md`]
 
 ## Implementation
 
-[link to react-* package folder]
-
-- [ ] Component implementation
+- [ ] [Component implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#implementation)
   - [link(s) to component implementation related PRs]
```

Comment:
I know you didn't change this part, but do we really need to ask for "link(s) to component implementation related PRs". I'd like to keep the epic lightweight, it seems too much of an ask for folks to come back and update this epic every time they have a new PR.

Reply:
I removed the lines asking for links to PRs, and added a reminder to link this Epic issue from any PRs made for it.